internal/nginx/manager.go

```diff
@@ -87,7 +86,7 @@ type LocalManager struct {
 	dhparamFilename              string
 	tlsPassthroughHostsFilename  string
 	verifyConfigGenerator        *verifyConfigGenerator
-	verifyClient                 *verifyClient
+	verifyClient                 *VerifyClient
```

Comment:
This won't be necessary if we use the Manager collector, another reason to use it.